config/prepare/manager_config_patch.yaml

```diff
@@ -2,7 +2,6 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: controller-manager
-  namespace: system
 spec:
```

Comment:
does it need the namespace to do the patch?

Reply:
Well done, another good spot! :)